includes/qaquery.php

```diff
@@ -501,13 +504,16 @@ function ap_recent_post_activity() {
  * Get last active time in human readable format.
  *
  * @param  mixed $post_id Post ID/Object.
- * @return string
+ * @return string|null
```

Comment:
@mebishalnapit You cannot change the return type like this as this function may be used in other places or other peoples template customizations. This function should always return a string. When no post return a string `Invalid post`.

Reply:
@rahularyan Modified to return `Invalid post` if no post is available.